scripts/run-rules.sh

```diff
@@ -22,7 +22,7 @@ OUTPUTDIR=$KB_DATA_DIR/$RULETYPE/
 RULES_RES_DIR=edu/ucdenver/ccp/kabob/build/$RULETYPE/
 
 echo Running rules
-mkdir --parents $OUTPUTDIR
+mkdir -p $OUTPUTDIR
```

Comment:
Yep, encountered it while trying to run a rule locally. Simple fix, so no worries.